package.json

```diff
@@ -44,9 +44,9 @@
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.0",
-    "@angular/compiler": "^2.0.0-rc.6",
+    "@angular/compiler": "^2.0.0-rc.7",
```

Comment:
I notice that CLI itself uses a ^ dependency, while the generated blueprint uses a fixed dependency. I like that letter fixed dependency, mostly because of so many past incidents. I am curious whether the floating dependency at the top level CLI package file could make it possible for bugs to slip past undetected, for example if a build were to be run in a situation where CLI and its test suite etc. are using version N+1 but the generated program will use version N.